api/illumio/cloud/k8scluster/v1/resource_config.proto

```diff
@@ -94,6 +95,185 @@ message KubernetesResourceMutation {
+// https://pkg.go.dev/github.com/cilium/cilium/api/v1/flow#readme-ipversion
+enum IPVersion {
+  IP_VERSION_IP_NOT_USED_UNSPECIFIED = 0;
+  IP_VERSION_IPV4 = 1;
+  IP_VERSION_IPV6 = 2;
+}
+
+// Information identifying Layer4 object in Cilium
+// https://pkg.go.dev/github.com/cilium/cilium/api/v1/flow#readme-layer4
+message Layer4 {
+  oneof protocol {
+    TCP tcp = 1;
+    UDP udp = 2;
+    // ICMP is technically not L4, but mutually exclusive with the above
+    ICMPv4 icmpv4 = 3;
+    ICMPv6 icmpv6 = 4;
+    SCTP sctp = 5;
+  }
+}
+
+// Information identifying an TCP object in Cilium
+// https://pkg.go.dev/github.com/cilium/cilium/api/v1/flow#readme-tcp
+message TCP {
+  uint32 source_port = 1;
+  uint32 destination_port = 2;
+  TCPFlags flags = 3;
+}
+
+// Information identifying possible TCPFlags in TCP message.
+// https://pkg.go.dev/github.com/cilium/cilium/api/v1/flow#readme-tcpflags
+message TCPFlags {
+  bool fin = 1;
+  bool syn = 2;
+  bool rst = 3;
+  bool psh = 4;
+  bool ack = 5;
+  bool urg = 6;
+  bool ece = 7;
+  bool cwr = 8;
+  bool ns = 9;
+}
+
+// Information identifying an UDP object in Cilium
+// https://pkg.go.dev/github.com/cilium/cilium/api/v1/flow#readme-udp
+message UDP {
+  uint32 source_port = 1;
+  uint32 destination_port = 2;
+}
+
+// Information identifying an SCTP object in Cilium
+// https://pkg.go.dev/github.com/cilium/cilium/api/v1/flow#readme-sctp
+message SCTP {
+  uint32 source_port = 1;
+  uint32 destination_port = 2;
+}
+
+// Information identifying an ICMPv4 object in Cilium
+// https://pkg.go.dev/github.com/cilium/cilium/api/v1/flow#readme-icmpv4
+message ICMPv4 {
+  uint32 type = 1;
+  uint32 code = 2;
+}
+
+// Information identifying an ICMPv6 object in Cilium
+// https://pkg.go.dev/github.com/cilium/cilium/api/v1/flow#readme-icmpv6
+message ICMPv6 {
+  uint32 type = 1;
+  uint32 code = 2;
+}
+
+// CiliumEventType from which the flow originated.
+// https://pkg.go.dev/github.com/cilium/cilium/api/v1/flow#readme-ciliumeventtype
+message CiliumEventType {
+  // type of event the flow originated from, i.e.
+  // github.com/cilium/cilium/pkg/monitor/api.MessageType*
+  int32 type = 1;
+  // sub_type may indicate more details depending on type, e.g.
+  // - github.com/cilium/cilium/pkg/monitor/api.Trace*
+  // - github.com/cilium/cilium/pkg/monitor/api.Drop*
+  // - github.com/cilium/cilium/pkg/monitor/api.DbgCapture*
+  int32 sub_type = 2;
+}
+
+// Information identifying Endpoint object in Cilium
+// https://pkg.go.dev/github.com/cilium/cilium/api/v1/flow#readme-endpoint
+message Endpoint {
+  uint32 id = 1;
+  uint32 identity = 2;
+  string cluster_name = 7;
+  string namespace = 3;
+  repeated string labels = 4;
+  string pod_name = 5;
+  repeated Workload workloads = 6;
+}
+
+// Information identifying NetworkInterface object in Cilium
+// https://pkg.go.dev/github.com/cilium/cilium/api/v1/flow#readme-networkinterface
+message NetworkInterface {
+  uint32 index = 1;
+  string name = 2;
+}
```

Comment:
That's not interesting to us a priori. Remove.
```suggestion
```

Reply:
Done